authenticating-users/main.py

```diff
@@ -18,28 +18,9 @@
 from flask import Flask
 app = Flask(__name__)
 
-CERTS = None
 AUDIENCE = None
 
 
-# [START getting_started_auth_certs]
```

Comment:
@Jipje can you let me know about why you're removing this block here? Thanks.

Reply:
(I answered my own question. I need to fix some stuff in documentation because removing this block will break a page. )